vm/actor/tests/common/mod.rs

```diff
@@ -76,6 +80,36 @@ pub struct ExpectedVerifySig {
     pub result: ExitCode,
 }
 
+#[derive(Clone, Debug)]
+pub struct ExpectVerifySeal {
+    seal: SealVerifyInfo,
+    exit_code: ExitCode,
+}
+
+#[derive(Clone, Debug)]
+pub struct ExpectVerifyPoSt {
+    post: WindowPoStVerifyInfo,
+    exit_code: ExitCode,
+}
+
+#[derive(Clone)]
+pub struct ExpectVerifyConsensusFault {
+    require_correct_input: bool,
+    block_header_1: Vec<u8>,
+    block_header_2: Vec<u8>,
+    block_header_extra: Vec<u8>,
+    fault: Option<ConsensusFault>,
+    exit_code: ExitCode,
+}
+
+#[derive(Clone)]
+pub struct ExpectComputeUnsealedSectorCid<'a> {
+    reg: RegisteredSealProof,
+    pieces: &'a Vec<PieceInfo>,
```

Comment:
you should never need to have a reference to a vec instead of using a slice, is there a reason this is taking a reference to this where other types take owned values?

Reply:
No lol I was trying to match the argument in the function that used pieces. Completely forgot slices were a thing [herp derp]